### PR TITLE
Renamed Texel Size node to Texture Size and added texel size outputs

### PR DESCRIPTION
**Pull request by bencloward**
Previously, the Texel Size node was actually returning the texture size - so I've renamed the node to reflect that.  I also added two new output ports to the node so that in addition to outputing the texture size, it also outputs the texel size.

# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [x] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Change the Texel Size node to Texture Size so that the name describes what the node actually does, and also expose the texel size output data so that the node also does what it used to be called.

---
### Testing status
Describe what manual/automated tests were performed for this PR

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Returns the **Width** and **Height** of the texel size of **Texture 2D** input. Uses the built in variable `{texturename}_TexelSize` to access special properties of a **Texture 2D**.
+Returns the **Width** and **Height** of the texture size of **Texture 2D** input as well as the **Width** and **Height** of the texel size. Uses the built in variable `{texturename}_TexelSize` to access special properties of a **Texture 2D**.
```
Comment (cdxntchou) [inline com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Returns the texel resolution of the Texture 2D input, as well as the size of each texel in the texture.

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md [hunk 2]
```diff
@@ -13,8 +13,10 @@ If you experience texture sampling errors while using this node in a graph which
 | Name        | Direction           | Type  | Binding | Description |
 |:------------ |:-------------|:-----|:---|:---|
 | Texture      | Input | Texture | None | Texture asset |
-| Width      | Output | Float    | None | Texel width |
-| Height | Output      |    Float    | None | Texel height |
+| Width      | Output | Float    | None | Texture width |
```
Comment (cdxntchou) [inline com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: should update the description to note the units returned:    Texture width in texels

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md [hunk 3]
```diff
@@ -13,8 +13,10 @@ If you experience texture sampling errors while using this node in a graph which
...
+| Height | Output      |    Float    | None | Texture height |
+| Texel Width      | Output | Float    | None | Texel width |
```
Comment (cdxntchou) [inline com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Texel width in UV coordinates

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md (hunk 1 above)
Comment (cdxntchou) [reply com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Do you think texel needs a definition?   Not sure how common that term is.

**Review by cdxntchou — APPROVED** [review]
Code looks good!   Just a few comments on the docs.

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md (hunk 2 above)
Comment (bencloward) [reply com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Made this change in the new version of the documentation that Sarah uploaded.

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md (hunk 3 above)
Comment (bencloward) [reply com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Made this change in the new version of the documentation that Sarah uploaded.

File: com.unity.shadergraph/Documentation~/Texel-Size-Node.md (hunk 2 above)
Comment (bencloward) [reply com.unity.shadergraph/Documentation~/Texel-Size-Node.md]: Made this change in the new version of the documentation that Sarah uploaded.

**Review by jessebarker — APPROVED** [review]
LGTM

File: com.unity.shadergraph/Documentation~/Texture-Size-Node.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+The Texture Size node is compatible with all render pipelines.
+
+## Ports
+
+| Name         | Direction | Type     | Binding | Description |
+|:------------ |:----------|:---------|:--------|:------------|
+| Texture      | Input     | Texture  | None    | The Texture 2D asset to measure. |
+| Width        | Output    | Float    | None    | The width of the Texture 2D asset in texels. |
+| Height       | Output    | Float    | None    | The height of the Texture 2D asset in texels.. |
```
Comment (sarah-welton) [inline com.unity.shadergraph/Documentation~/Texture-Size-Node.md:26]: There's an extra period here :) Make sure you delete that!

**Review by sarah-welton — CHANGES_REQUESTED** [review]
Only one minor thing, that extra period. I'm not currently equipped to go into your branch and actually see the node, but please consider updating the "TexelSizeNodeThumb.png" under Documentation~/images (is used on Input-Nodes.md).

File: com.unity.shadergraph/Documentation~/Texture-Size-Node.md (hunk 4 above)
Comment (bencloward) [reply com.unity.shadergraph/Documentation~/Texture-Size-Node.md]: Done and done.

**Pull request merged**